data/Hishammpsnhn.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Mohammed Hisham",
+  "type": "personal",
+  "bio": "Self-taught js developer",
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mohammad-hisham-b14296228/",
+      "icon": "FaLinkedin"
+    }
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/_hisham_mp/",
+      "icon": "FaInstagram"
+    }
```

Comment:
```suggestion
    },
```

Reply:
I have marked this as `unresolved` as the change needs to be accepted for the json file to be valid